### PR TITLE
Update bug-report--english-.md

### PR DESCRIPTION
**Pull request by RishiM1408**
I'm facing this issue with my submission

### CODE REVIEW

**Comment by RishiM1408** [comment]
Please let me know if this gets resolved.

**Comment by Rajatsinha05** [comment]
would you like to explain more

**Comment by desaimann37** [comment]
Okay Sure!!!

**Comment by desaimann37** [comment]
Here I have Done Mistake Which I felt not but compiler is giving me error when we write overridden function call and returning statement so for soln go to my home page of leetcode which is @Mann_12345 and checkout recent submissions for the above ques , Thank you...

File: .github/ISSUE_TEMPLATE/bug-report--english-.md [hunk 1]
```diff
@@ -1,8 +1,8 @@
 ---
-name: Bug report (English)
-about: Create a bug report to help us improve our content.
```
Comment (zuhriddinzayniddinov) [inline .github/ISSUE_TEMPLATE/bug-report--english-.md:3]: +++++

**Comment by RishiM1408** [comment]
Made it work a while ago, Thanks for immaculate response.

**Pull request closed without merging**